### DISCUSSION TITLE
Clickable id in row's

### DISCUSSION THREAD

**Discussion by matthewrichter** [Power-Components/livewire-powergrid#929] [Q&A] [answered]
Hi 
Hope everyone is well.
I am currently using Powergrid in one of my projects. 

Are you able to have a clickable ID in a row that goes to a route?
So for example if 5 records load in the table and the column name is id, make each row "id" clickable to a laravel route.

Any help would be greatly appreciated.

Thanks
Matt

**Comment by abhaypithadiya** [comment, marked as answer]
Hey Matt!

Try this out, I am not sure if this is exactly what you are are trying to get.

```
    public function addColumns(): PowerGridEloquent
    {
        return PowerGrid::eloquent()
        ->addColumn('id_formatted', function (Model $model) {
            return '<a href="'.route('your_link_name_here', $model->id).'">'.$model->id.'</a>';
        })
    }
```

```
    public function columns(): array
    {
        return [
            Column::make('ID', 'id_formatted','id')
            ->sortable(),
    }
```

Hope this helps!

**Reply by matthewrichter** [reply]
Thanks for that, worked perfect!
